### DISCUSSION TITLE
Is it possible to make daisy ui use the exact colour instead of changing to a difference hex?

### DISCUSSION THREAD

**Discussion by DevOfManyThings** [saadeghi/daisyui#1527] [Q&A] [answered]
Eg in my daisyui config I have primary configured as #0FBA81, however when I use `btn-primary` it comes through as #0FB880 in the browser (tested both day/night mode in chrome/firefox). It's a minor color discrepancy but it's annoying if something is being designed with pre-decided branding and can't use the exact hex code.

I tried looking through the source code and I think because of how daisy converts the hex to rgb this might be unavoidable? But maybe I've just missed something?

**Comment by saadeghi** [comment]
The reason is when we change the Hex code to HSL values, sometimes the numbers have decimal places. For example, if it's `24.31` we round it to `24`. The difference is not recognizable by eye but the Hex code is slightly different.  
I did this just for the HSL values of CSS variables to look cleaner but apparently that wasn't a good decision.

This will be fixed soon.

**Comment by saadeghi** [comment, marked as answer]
Please update to the latest version. It is fixed now in version `2.47.0`
